Source/Meadow.Contracts/Hardware/Contracts/IPinDefinitions.cs

```diff
@@ -28,6 +28,6 @@ public interface IPinDefinitions : IEnumerable<IPin>
         /// <summary>
         /// Gets or sets the IPinController associated with the IPins
         /// </summary>
-        IPinController Controller { get; set; }
+        IPinController? Controller { get; set; }
```

Comment:
Does this impact anything in Meadow.Core?

Reply:
@adrianstevens Good question, I was refactoring for Meadow.CLI. I'll load up Meadow.Core to see what it throws. Thanks for pointing it out.

Reply:
As per Slack "Meadow.Core builds without errors or warnings, so in theory Meadow.Core has code to check it’s Controller's nullability.